apps/passport/app/root.tsx

```diff
@@ -44,6 +45,9 @@ import type { FLASH_MESSAGE } from './utils/flashMessage.server'
 
 import { getFlashSession, commitFlashSession } from './session.server'
 
+import { PostHogProvider } from 'posthog-js/react'
+import posthog from 'posthog-js'
```

Comment:
```suggestion
import posthog from 'posthog-js'
import { PostHogProvider } from 'posthog-js/react'
```

I find this easier for readability.